sdk/monitor/azure-monitor-query/azure/monitor/query/_models.py

```diff
@@ -73,12 +73,15 @@ class LogsQueryResult(object):
     :paramtype statistics: object
     :keyword render: Any object.
     :paramtype render: object
+    :keyword error: Any object.
+    :paramtype error: object
```

Comment:
is this more of a result object or would a user create this in code? I ask because I think it would make more sense to document these as `ivars` if it's just a result object (edit: same for the type below)

Reply:
fair enough - updating